x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_landing/translations.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const LANDING_TITLE = i18n.translate(
+  'xpack.integrationAssistant.createIntegrationLanding.title',
+  {
+    defaultMessage: 'Create new integration',
+  }
+);
+
+export const LANDING_DESCRIPTION = i18n.translate(
+  'xpack.integrationAssistant.createIntegrationLanding.description',
+  {
+    defaultMessage:
+      'Start an AI-driven process to build your integration step-by-step, or you can either upload a .zip package with an already created integration',
```

Comment:
```suggestion
      'Start an AI-driven process to build your integration step-by-step, or upload a .zip package with an existing integration',
```

Reply:
I like the change. However, "existing integration" is not entirely correct, the integration should not exist in the environment where the zip is uploaded. The package creation/installation will happen in one environment and the zip will be used to deploy the same integration to other environments, where it does not (yet) exist. What about this:

`Start an AI-driven process to build your integration step-by-step, or upload a .zip package of a previously created integration`

Reply:
https://github.com/elastic/kibana/pull/186304/commits/f2fe6207796026eb8d3b935b3270be3a43f7c9c2